README.md

```diff
@@ -59,22 +74,34 @@ class PythonSlave(Fmi2Slave):
 ###### Create the FMU 
 
 ```
-java -jar pythonfmu-builder.jar -f pythonslave.py pythonfmu
+pythonfmu-builder -f pythonslave.py pythonfmu
 ```
 
 In this example a python class named `PythonSlave` that extends `Fmi2Slave` is declared in a file named `pythonslave.py`. 
-`pythonfmu` is a folder containing additional project files required by the python script, including `fmi2slave.py`. 
+`pythonfmu` is a optional folder containing additional project files required by the python script. 
 Project folders such as this will be recursively copied into the FMU. Multiple project files/folders may be added.
 
 
+###### Use the FMU
```

Comment:
I feel perhaps this is more suited in a Wiki page. A bit much commercial for `pyfmi`, when any FMI tools would do.

Reply:
Make sense